.vscode/settings.json

```diff
@@ -7,5 +7,6 @@
   "eslint.nodePath": "./node_modules",
   "editor.codeActionsOnSave": {
     "source.fixAll": true
-  }
+  },
+  "jest.jestCommandLine": "yarn test"
```

Comment:
What does this do?

Reply:
It makes sure that the Jest vscode extension uses yarn test instead of using the jest binary.